app/assets/stylesheets/components/banner.scss

```
@@ -142,3 +142,56 @@
   }
 
 }
+
+.banner-dashboard {
+
+  $baseline-shift: 5px;
+
+  display: block; // for browsers that don't support flexbox
+  display: flex;
+  align-items: baseline;
+  flex-wrap: wrap;
+  padding: $gutter-two-thirds 0;
```

Needs the spacing tweaking by a fraction to accommodate the baseline shift 
```suggestion
  padding: ($gutter-two-thirds - 1px) 0 ($gutter-two-thirds + 1px) 0;
```